regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
+                         "field=gov_agency&dependentOnValue=Federal")
+    response = requests.get(
+        url,
+        headers={'X-Api-Key': settings.REGS_GOV_API_KEY}
+    )
+    response.raise_for_status()
```

Comment:
If this raises an error, our app throws a 500, right? Should we catch and return a different status code? Even if 500 is appropriate, a json-formatted 500 would be nice (iirc this will return an html-formatted 500, but my memory could be off).